### DISCUSSION TITLE
[Freature Request] Mark games on site based on current progress

### DISCUSSION THREAD

**Discussion by Blazekickn** [RetroAchievements/RAWeb#1125] [Ideas]
![image](https://user-images.githubusercontent.com/38176599/190972511-31f6b138-a586-41fc-bc38-ac27318583c9.png)

Before v2 broke it, there was a Xymjak script I and many others used a ton - RA_ColorMarkGames [(found here)](https://retroachievements.org/viewtopic.php?t=12821). Basically it changed the color of game names on console lists, hubs, and the forums depending on your progress in the set (by default, Yellow=Unplayed, Light Blue=Less than 50%, Dark Blue=More than 50%, Grey=100%). It was an extremely useful quality of life for quickly identifying which games I have and haven't played. Unfortunately Xymjak is mostly inactive and can't really update the script anymore, so I'd like to see something similar to this officially added on the site.

**Comment by Jamiras** [comment]
Already requested in #697. I see the solution as adding a percentage column (or maybe a progress bar column). Xymjak's script scraped the progress from the user's profile page and stored it in the client's browser cache. There is some non-negligible overhead in calculating the user progress for each game. This is one of the features that would benefit from denormalized data (pre-calculated progress), which is a promise of the post-v2 framework upgrade.

**Comment by Jamiras** [comment]
A Progress column is now available on game lists.
